packages/email-field/src/vaadin-email-field.js

```diff
@@ -49,6 +49,15 @@ export class EmailField extends TextField {
     return 'vaadin-email-field';
   }
 
+  static get properties() {
+    return {
+      pattern: {
+        type: String,
+        value: '^([a-zA-Z0-9_\\.\\-+])+@[a-zA-Z0-9-.]+\\.[a-zA-Z0-9-]{2,}$',
```

Comment:
**note:** Moved the default value for `pattern` here in order to allow its unset by passing an empty `pattern` attribute.